### PR TITLE
Fix download-cmd parameter about adding nodes and updating cluster

### PR DESCRIPTION
**Pull request by Forest-L**
Signed-off-by: Forest-L <lilin@yunify.com>

### CODE REVIEW

**Review by pixiake — COMMENTED** [review]
/lgtm

File: go.mod [hunk 1]
```diff
@@ -4,7 +4,7 @@ go 1.14
 
 require (
 	github.com/dominodatalab/os-release v0.0.0-20190522011736-bcdb4a3e3c2f
-	github.com/go-logr/logr v0.4.0
+	github.com/go-logr/logr v0.1.0
```
Comment (LinuxSuRen) [inline go.mod:7]: It's wired. Why downgrade the version of `go-logr` here.

Comment (pixiake) [reply go.mod:7]: operator-sdk does not support logr v0.4.0

Comment (LinuxSuRen) [reply go.mod:7]: Got you.

**Pull request merged**